components/library/Timeline/Preview/Timeline.jsx

```diff
@@ -0,0 +1,91 @@
+import { BlockStack, Box, Icon, InlineGrid, InlineStack, Link, Text } from "@shopify/polaris";
+import { AlertCircleIcon, CheckCircleIcon, ChevronRightIcon, CircleChevronRightIcon } from "@shopify/polaris-icons";
+import styles from './timeline.module.css';
+
+export default function Timeline({ items }) {
+
+  function getBulletIconFromTone(tone) {
+    switch (tone) {
+      case 'critical':
+      case 'caution':
+        return AlertCircleIcon;
+      case 'success':
+        return CheckCircleIcon
+      case 'base':
+      case undefined:
+        return null;
+      default:
+        return CircleChevronRightIcon;
+    }
+  }
+
+  let lastDate = null;
+
+  return (
+    <Box>
+      <BlockStack gap="300">
+        {items?.length ? items.map((item, index) => {
+          const currentDate = item.timestamp.toLocaleDateString([], { year: 'numeric', month: 'long', day: 'numeric' });
+          const showDate = currentDate !== lastDate;
+          lastDate = currentDate;
+          const bulletIcon = getBulletIconFromTone(item.tone);
+
+          return (
+            <>
+              {showDate && (
+                <InlineGrid key={index} gap="200" columns="30px auto 90px">
+                  <div>
+                    &nbsp;
+                  </div>
+                  <BlockStack gap="0">
+                    <Box paddingBlockStart="600">
+                      <Text alignment="start" as="h2" variant="bodySm" tone="subdued" key={`date-${index}`}>
+                        {currentDate}
+                      </Text>
+                    </Box>
+                  </BlockStack>
+                  <div>&nbsp;</div>
+                </InlineGrid>
+
+              )}
+
+              <InlineGrid key={index} gap="200" columns="30px auto 90px" alignItems="center">
+                <div className={styles['timeline-icon']}>
+                  {item.tone === 'base' || !bulletIcon ? (
+                    <div className={styles['timeline-icon-base']}>
+                      <div className={styles['timeline-icon-base-inner']} />
+                    </div>
+                  ) : (
+                    <span className={styles['timeline-icon-polaris-icon']}><Icon source={bulletIcon} tone={item.tone} /></span>
+                  )}
+                </div>
+                <Box className={styles['timeline-event-description']}>
+                  <InlineStack gap="200" wrap={false} blockAlign="center">
+                    {item.icon && (
+                      <Box>{item.icon}</Box>
```

Comment:
Wrapping this icon in a `Box` causes it to not align properly:

![CleanShot 2024-09-21 at 11 12 10@2x](https://github.com/user-attachments/assets/18c62e7a-7d2e-436a-9a96-429937469a12)

You could just do this to render the icon if it exists and it'll be aligned properly.

```
{item.icon}
{item.url ? (
...
```